onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -153,6 +153,15 @@ $ python3 -m models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --output l
 $ python3 -m onnxruntime.transformers.models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --output llama2-7b-int4-cpu --precision int4 --quantization_method blockwise --execution_provider cpu
 ```
 
+Export Sharded model, llama-70b into 4 partitions
+```
+# From source:
+$ 1. Get OnnxRuntime code from https://github.com/frankdongms/transformers/tree/frdong/shard_llama or
+$    wait until PR: https://github.com/huggingface/transformers/pull/27119 got merged into HF transformers
```

Comment:
We can make a PR later to update `requirements.txt` to use `transformers` from source. Let's remove this line for now.